Components documentation improvements (#2121)
withastro/starlight: docs/src/content/docs/components/code.mdx

---
title: Code

> description: Learn how to display syntax highlighted code using the Code component.

Reviewer: I feel like the main reason you'd use this is when you can't use the Markdown code block, so worth mentioning it in the description?
```suggestion
description: Learn how to display syntax highlighted code in Starlight without Markdown code blocks.
```
